script.js

```diff
@@ -0,0 +1,48 @@
+const container = document.querySelector("#container");
+const resetBtn = document.querySelector("#reset");
+const changeSizeBtn = document.querySelector("#change-size");
+
+const createGrid = (size) => {
+  const cellSize = 600 / size;
```

Comment:
`600` as number is little confusing, please try to add comment saying what the number `600` present or take variable constant with meaningful naming convention and then use it here